src/pages/Mainpage/Mainpage.js

```diff
@@ -0,0 +1,107 @@
+import { useEffect, useState } from "react";
+
+import MainIcon from "../../components/mainIcon/MainIcon";
+
+import background_image from "../../assets/background_image.jpg";
+import { MainpageContainer } from "./Mainpage.element";
+import Card from "../../components/card/Card";
+
+const Mainpage = ({ clickedIcon, setClickedIcon }) => {
+  const [mainIconPosition, setMainIconPosition] = useState({
+    top: 30,
+    left: 30,
+  });
+  const [textIconPosition, setTextIconPosition] = useState({
+    top: 160,
+    left: 30,
+  });
+
+  useEffect(() => {
+    setMainIconPosition({
+      top: 30,
+      left: 30,
+    });
+    setTextIconPosition({
+      top: 160,
+      left: 30,
+    });
+  }, []);
```

Comment:
`mainIconPosition`과 `textIconPosition` 모두 `useState` 훅 사용 시 초깃값을 설정해 주고 계신데, `useEffect`로 한 번 더 초깃값을 설정할 필요가 있을까요?.?

이렇게 수정해 봐도 좋을 것 같습니다!

```suggestion
  const [iconPosition, setIconPosition] = useState({
  main: {
    top: 30,
    left: 30,
  },
  text: {
    top: 160,
    left: 30,
  },
});
```

Reply:
앗... 여기 비하인드 스토리가 있어요 ㅎㅎ
원래는 drag and drop 기능을 넣어서 drop한 위치를 state에 저장해서 관리하려고 했는데 기능 구현을 실패하는 바람에 setIconPosition이 쓰이질 않더라고요. CI = false env를 넣어주면 되긴하는데 ~~(그냥 노랑 warning이 콘솔창에 뜨는게 싫어서)~~ 나중에 setIconPosition을 추후에 사용할 경우를 생각해서 넣어주었습니다.